src/test/java/apoc/export/cypher/ExportCypherTest.java

```diff
@@ -16,34 +15,18 @@
 
 import static apoc.util.MapUtil.map;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
 
 /**
  * @author mh
  * @since 22.05.16
  */
 public class ExportCypherTest {
 
-    private static final String EXPECTED = String.format("begin%n" +
-            "CREATE (:`Foo`:`UNIQUE IMPORT LABEL` {`name`:\"foo\", `UNIQUE IMPORT ID`:0});%n" +
-            "CREATE (:`Bar` {`name`:\"bar\", `age`:42});%n" +
-            "CREATE (:`Bar`:`UNIQUE IMPORT LABEL` {`age`:12, `UNIQUE IMPORT ID`:2});%n" +
-            "commit%n" +
-            "begin%n" +
-            "CREATE INDEX ON :`Foo`(`name`);%n" +
-            "CREATE CONSTRAINT ON (node:`Bar`) ASSERT node.`name` IS UNIQUE;%n" +
-            "CREATE CONSTRAINT ON (node:`UNIQUE IMPORT LABEL`) ASSERT node.`UNIQUE IMPORT ID` IS UNIQUE;%n" +
-            "commit%n" +
-            "schema await%n" +
-            "begin%n" +
-            "MATCH (n1:`UNIQUE IMPORT LABEL`{`UNIQUE IMPORT ID`:0}), (n2:`Bar`{`name`:\"bar\"}) CREATE (n1)-[:`KNOWS`]->(n2);%n" +
-            "commit%n" +
-            "begin%n" +
-            "MATCH (n:`UNIQUE IMPORT LABEL`)  WITH n LIMIT 20000 REMOVE n:`UNIQUE IMPORT LABEL` REMOVE n.`UNIQUE IMPORT ID`;%n" +
-            "commit%n" +
-            "begin%n" +
-            "DROP CONSTRAINT ON (node:`UNIQUE IMPORT LABEL`) ASSERT node.`UNIQUE IMPORT ID` IS UNIQUE;%n" +
-            "commit");
+    private static final String EXPECTED_NODES = String.format("begin%nCREATE (:`Foo`:`UNIQUE IMPORT LABEL` {`name`:\"foo\", `UNIQUE IMPORT ID`:0});%nCREATE (:`Bar` {`name`:\"bar\", `age`:42});%nCREATE (:`Bar`:`UNIQUE IMPORT LABEL` {`age`:12, `UNIQUE IMPORT ID`:2});%ncommit");
+
+    private static final String EXPECTED_RELATIONSHIPS = String.format("begin%nMATCH (n1:`UNIQUE IMPORT LABEL`{`UNIQUE IMPORT ID`:0}), (n2:`Bar`{`name`:\"bar\"}) CREATE (n1)-[:`KNOWS`]->(n2);%ncommit");
+
+    private static final String EXPECTED_SCHEMA = String.format("begin%nCREATE INDEX ON :`Foo`(`name`);%nCREATE CONSTRAINT ON (node:`Bar`) ASSERT node.`name` IS UNIQUE;%nCREATE CONSTRAINT ON (node:`UNIQUE IMPORT LABEL`) ASSERT node.`UNIQUE IMPORT ID` IS UNIQUE;%ncommit%nschema await%nbegin%nMATCH (n:`UNIQUE IMPORT LABEL`)  WITH n LIMIT 20000 REMOVE n:`UNIQUE IMPORT LABEL` REMOVE n.`UNIQUE IMPORT ID`;%ncommit%nbegin%nDROP CONSTRAINT ON (node:`UNIQUE IMPORT LABEL`) ASSERT node.`UNIQUE IMPORT ID` IS UNIQUE;%ncommit");
```

Comment:
Do you mean the "drop" statement should be done at the end of all the other statements?